app/controllers/individual/WhichCountryTaxForIndividualController.scala

```diff
@@ -112,6 +120,12 @@ class WhichCountryTaxForIndividualController @Inject()(
     }
   }
 
+  private def dynamicGuidance(index: Int): String =
```

Comment:
Is there any value in moving these methods into a journey helper that takes an index and we pass the two strings we want to use? As to not repeat code?